README.md

```diff
@@ -4,16 +4,67 @@
 
 ## Usage
 
+Formulator is a library to give you:
```

Comment:
"Formulator is a library for *Phoenix* that gives you:"

Since I think this is only for Phoenix since it's tied to the Phoenix HTML helpers